mm2src/mm2_net/src/wasm_event_stream.rs

```diff
@@ -17,7 +19,8 @@ pub async fn handle_worker_stream(ctx: MmArc) {
             "message": event.message(),
         });
 
-        let worker = web_sys::Worker::new("worker.js").expect("Missing worker.js");
+        let worker_path = config.worker_path.as_deref().unwrap_or(DEFAULT_WORKER_PATH);
+        let worker = web_sys::Worker::new(worker_path).expect("Missing worker.js");
```

Comment:
It might not be "worker.js".

Reply:
Was already fixing this. Done :)

Reply:
we might have another worker.js that serves a diff purpose In the future so why not make it `balance_streaming_worker.js` as default ?

Reply:
I left it as worker.js for backward compatibility with this https://github.com/KomodoPlatform/react-komodefi-wasm/tree/streaming ref https://github.com/KomodoPlatform/komodo-defi-framework/pull/1978#issue-1918819783

Reply:
I can make it `balance_streaming_worker.js` if agreed by @onur-ozkan and they would have to fix it in the `react-komodefi-wasm` branch

> Also, worker.js is a very basic worker that's doing much and we might have a more complex worker in the future

Can you please elaborate on this @borngraced ?

Reply:
I have no objections regarding to changing the filename of the worker file.

Reply:
> Also, worker.js is a very basic worker that's doing much and we might have a more complex worker in the future

I believe we should need some feedback from frontend devs. They may prefer to handle all the events from single worker file by filtering the incoming data.

Reply:
> I believe we should get some feedback from the frontend devs. They may prefer to handle all the events from single worker file by filtering the incoming data.

It's still a single worker file for event streaming

Reply:
If they requested multiple workers in the future we can do it in another PR.

Reply:
> the current worker script used for balance streaming is only logging the messages to the console..

It's just for demonstration, it doesn't have to be that way obviously.

You can handle all kinds of operations from a single worker file by filtering the incoming payload. As I mentioned, this preference should be defined by the frontend developers not by us as this affects their project structure directly.

Reply:
> It's still a single worker file for event streaming

I am aware of it, I meant event streaming + different tasks on workers

Reply:
> I am aware of it, I meant event streaming + different tasks on workers

In this case they can change the config I guess.